### ISSUE TITLE
Expo 52 has a broken version of Lottie

### ISSUE THREAD

**Issue by howard-bailey-nova** [expo/expo#32956]
### Minimal reproducible example

https://github.com/howard-bailey-nova/expo-52-lottie-repro

### What platform(s) does this occur on?

iOS

### Where did you reproduce the issue?

in Expo Go

### Summary

After upgrading to Expo 52, `lottie-react-native` (included as part of Expo) animations no longer fire consistently on iOS. Lottie already has a fix for this but the fixed version `7.1.0` is not included in Expo 52 (instead version `7.0.0`) is.

We principally use Expo Go at the moment, and although we could install the new version of lottie manually and create a development build it would be a major distraction, so if a new Expo Go version including this fix could be pushed we would be very grateful.

We really appreciate any help you are able to give.

Original issue in Lottie: https://github.com/lottie-react-native/lottie-react-native/issues/1257

### Environment

```text
  expo-env-info 1.2.1 environment info:
    System:
      OS: macOS 14.1
      Shell: 5.9 - /bin/zsh
    Binaries:
      Node: 22.10.0 - ~/.nvm/versions/node/v22.10.0/bin/node
      Yarn: 1.22.22 - ~/.nvm/versions/node/v22.10.0/bin/yarn
      npm: 10.9.0 - ~/.nvm/versions/node/v22.10.0/bin/npm
    Managers:
      CocoaPods: 1.15.2 - /opt/homebrew/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 23.4, iOS 17.4, macOS 14.4, tvOS 17.4, visionOS 1.1, watchOS 10.4
    IDEs:
      Android Studio: 2024.1 AI-241.15989.150.2411.11948838
      Xcode: 15.3/15E204a - /usr/bin/xcodebuild
    npmPackages:
      expo: ~52.0.7 => 52.0.7 
      expo-router: ~4.0.5 => 4.0.5 
      react: 18.3.1 => 18.3.1 
      react-dom: 18.3.1 => 18.3.1 
      react-native: 0.76.2 => 0.76.2 
      react-native-web: ~0.19.13 => 0.19.13 
    Expo Workflow: managed
```

### Expo Doctor Diagnostics

```text
Enabled experimental React Native Directory checks. Unset the EXPO_DOCTOR_ENABLE_DIRECTORY_CHECK environment variable to disable this check.
✔ Check package.json for common issues
✔ Check Expo config for common issues
✔ Check native tooling versions
✔ Check if the project meets version requirements for submission to app stores
✔ Check for common project setup issues
✔ Check dependencies for packages that should not be installed directly
✔ Check for app config fields that may not be synced in a non-CNG project
✔ Check for issues with Metro config
✔ Check npm/ yarn versions
✔ Validate packages against React Native Directory package metadata
✔ Check Expo config (app.json/ app.config.js) schema
✔ Check that packages match versions required by installed Expo SDK
✔ Check that native modules do not use incompatible support packages
✔ Check for legacy global CLI installed locally
✔ Check that native modules use compatible support package versions for installed Expo SDK

Didn't find any issues with the project!
```

**Comment by brentvatne** [comment]
@alanjhughes will update lottie for sdk 52, and this will be included in our next release of expo go. in the meantime you can use any version of lottie that you like using development builds. the go app is just a sandbox that has a fixed native runtime, so you can't change versions like this. however, you can do anything you like in a development builds: https://expo.dev/blog/expo-go-vs-development-builds

**Comment by RyanTG** [comment]
Test `"lottie-react-native": "7.1.0",` in a development build and for me it indeed fixed the issue. So yeah, would be good to have that as the recommended package for SDK 52.

**Comment by alanjhughes** [comment]
Resolved by #32969
